web/src/features/charts/NotEnoughDataMessage.tsx

```diff
@@ -0,0 +1,12 @@
+import { ChartTitle } from './ChartTitle';
+
+export function NotEnoughDataMessage({ translationKey }: { translationKey: string }) {
+  return (
+    <div className="w-full">
+      <ChartTitle translationKey={translationKey} hasLink={false} />
+      <div className="my-2 rounded bg-gray-200 py-4 text-center text-sm dark:bg-gray-800">
+        <p>Not enough data available to display chart</p>
```

Comment:
It should be possible to translate this string.

Reply:
You are absolutely right, I have made it possible and added translations for the most common languages :)